README.md

```diff
@@ -1,24 +1,58 @@
-# gender-bias-web
+# Reading For Gender Bias - FrontEnd (GEL team)
+## Table of contents 
+* [Introduction](#Introduction)
+* [Technologies](#Technologies)
+* [Installation and Usage](#Installation) 
+* [Credits](#Credits) 
 
-## Project setup
+
+## Introduction
+    This project is a web-based user interface for [Reading for Gender Bias](https://github.com/glam-lab/gender-bias). 
+    This project is currently under active development by member of the GLAM Lab at Whitman College to create an accessible tool. 
+
+## Technologies
+* [Vue Framework](https://vuejs.org/v2/guide/#:~:text=Vue%20(pronounced%20%2Fvju%CB%90%2F%2C,up%20to%20be%20incrementally%20adoptable.)
+* [Puppeteer, mocha, chai](https://medium.com/@ankit_m/ui-testing-with-puppeteer-and-mocha-part-1-getting-started-b141b2f9e21) for testing. 
+* [yarn](https://classic.yarnpkg.com/en/docs/)/[npm](https://docs.npmjs.com/) as package managers 
+
+## Installation and Usage
+### Backend installation process 
 ```
-yarn install
+git clone https://github.com/gender-bias/gender-bias
+cd gender-bias
+pip3 install -e .
 ```
+> NOTE: The last line in the above snippet installs this library in "editable" mode, which is probably fine while the library is in a state of flux.
 
-### Compiles and hot-reloads for development
+### Package Managers to run the project 
+To set up the project, install a package manager such as yarn or npm.
 ```
-yarn serve
+yarn install
+```
+Or 
 ```
+npm install
+``` 
+### Running the front-end 
+Once installed, add install Vue, and use yarn to open the server locally. Go to the right directory, and start the server. 
 
-### Compiles and minifies for production
 ```
+yarn add Vue / npm install vue
```

Comment:
The right way to install vue is listed [in the docs](https://cli.vuejs.org/guide/installation.html) as the following:

```shell
npm install -g @vue/cli
# OR
yarn global add @vue/cli
```


...but I don't think this step is necessary anyway, if the user has already run `[yarn|npm] install`.

Reply:
I think @j6k4m8 is saying to delete this instruction, because it's unnecessary. Since `vue` is listed in the `yarn.lock` file, it should be installed automatically when you run `yarn install`.

This makes me wonder if we should delete references to `npm` and just recommend `yarn`. Thoughts, @j6k4m8 ?

Reply:
Either works for me. I usually tend to support both but I don't have preferences here.

Reply:
We weren't really supporting `npm` so I removed the references.